.claude/agents/pr-scope-checker.md

```
@@ -0,0 +1,259 @@
+3. `origin/main` fallback
+
+```bash
+BASE=$(git symbolic-ref refs/remotes/origin/HEAD 2>/dev/null | sed 's|refs/remotes/origin/||' || echo "main")
+git diff $(git merge-base HEAD origin/$BASE)..HEAD --name-only
+```
+
+### 2. Domain Layer 매핑
+
+변경 파일을 아래 layer로 분류:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)